src/graphics/components/UIText.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+** EPITECH PROJECT, 2024
+** Stellar-Forge components
+** File description:
+** UIText.cpp
+**/
+
+#include "UIText.hpp"
+
+UIText::UIText(IObject* owner, const char *textStr, int size, sf::Color color, const char *fontPath)
+    : AGraphicsComponent(owner), textString(textStr), fontSize(size), color(color), fontPath(fontPath)
+{
+    if (std::string(fontPath) == "default") {
+        #ifdef _WIN32
+            fontPath = "C:\\Windows\\Fonts\\arial.ttf";
+        #elif defined(__linux__)
+            fontPath = "/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf";
+        #else
+            throw GraphicsException("Unknown OS: No default font available");
+        #endif
```

Comment:
![image](https://github.com/user-attachments/assets/dd2288c8-cf88-4a6f-959c-f5a943b3654b)

Reply:
Maybe we can put that in another place :joy: